src/asqi/workflow.py

```diff
@@ -414,10 +460,28 @@ def execute_single_test(
     result.container_output = container_result["output"]
     result.error_message = container_result["error"]
 
+    test_results = {}
+    technical_reports = []
+
     # Parse JSON output from container
     if container_result["success"]:
         try:
-            result.test_results = parse_container_json_output(result.container_output)
+            parsed_container_results = parse_container_json_output(
+                result.container_output
+            )
+            test_results, technical_reports = extract_container_json_output_fields(
+                parsed_container_results
+            )
+            result.test_results = test_results
+            result.technical_reports = technical_reports
+
+            host_volume = test_params.get("volumes", {}).get("output")
+            if host_volume:
+                for r in technical_reports:
+                    if r.get("report_path", "").startswith(str(OUTPUT_MOUNT_PATH)):
+                        r["report_path"] = r["report_path"].replace(
+                            str(OUTPUT_MOUNT_PATH), host_volume, 1
+                        )
```

Comment:
This is a little fragile e.g. - `/output/logs/output/report.html`. Do you also want to abstract it to a function e.g. `translate_report_path` so the intent is clearer